Podfile

```diff
@@ -1,4 +1,5 @@
-source 'https://github.com/CocoaPods/Specs.git'
+source 'https://cdn.cocoapods.org/'
+source 'git@github.com:mercadolibre/mobile-ios_specs.git'
```

Comment:
we cant use private pods

Reply:
I see. Let me figure out something else, without using the CP